db/migrate/20250409194908_add_failure_to_bpds_submission_status_enum.rb

```diff
@@ -0,0 +1,63 @@
+class AddFailureToBpdsSubmissionStatusEnum < ActiveRecord::Migration[7.2]
+  def up
+    add_enum_value :bpds_submission_status, 'failure'
+  end
+
+  def down
+    # Retrieve list of all successful and failed submissions and attempts
+    submissions = []
+    # Uncomment after implementing the Bpds::Submission model
+    # Bpds::Submission.pluck(:id, :latest_status)
+    attempts = []
+    # Uncomment after implementing the Bpds::SubmissionAttempt model
+    # Bpds::SubmissionAttempt.pluck(:id, :status)
+
+    # Drop the enum list and remove columns using the enum
+    safety_assured do
+      remove_column :bpds_submissions, :latest_status
```

Comment:
You'll first need to make a PR (and get it deployed) to ignore the column: https://github.com/ankane/strong_migrations?tab=readme-ov-file#removing-a-column